test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -77,6 +77,13 @@ public IActionResult GetContentResultWithContentTypeAndEncoding()
             return Content("content", "application/json", Encoding.ASCII);
         }
 
+        public IActionResult GetObjectResultWithNoContent()
+        {
+            var result = new ObjectResult(null);
+            result.StatusCode = 200;
```

Comment:
Use something other than 200. This is the default, so it doesn't make a great test